CustomTabBar/TabButton.swift

```diff
@@ -40,6 +40,11 @@ class TabButton: UIView {
             updateInsets()
         }
     }
+    var titleFont: UIFont? = nil {
+        didSet {
+            setupViews()
```

Comment:
This is still too intense. You just need to update the font if it changes, not reset all the views.